apps/api/src/routes/application-router.ts

```diff
@@ -44,4 +45,39 @@ applicationRouter.post('/application/edit', jsonParser, async (req, res) => {
 	}
 });
 
+// TODO: - Refactor endpoint logic once validation/dto flow is in place
+//       - verify if user can access applications
+//       - validate queryParam options using zod
+applicationRouter.get('/applications', async (req: Request<{}, {}, {}, any>, res) => {
+	const { userId, state, page, pageSize } = req.query;
+
+	//  Temporary userId check until validation/dto flow is confirmed
+	//  - reflect changes in swagger once refactored
+	if (!userId) {
+		res.status(400).send({ message: 'User Id is required' });
+		return;
+	}
+
+	// Check if sort exists and parse it if true
+	let sort = [];
+
+	if (req.query.sort) {
+		sort = JSON.parse(req.query.sort);
+	}
```

Comment:
Potential refactoring to:
```
const { userId, state, page, pageSize, sort: sortQuery } = req.query;

// Check if sort exists and parse it if true
const sort = !!sortQuery ? JSON.parse(sortQuery) : [];

const result = await getAllApplications({
		userId,
		state,
		sort,
		page: parseInt(page),
		pageSize: parseInt(pageSize),
	});
```
My reasoning being that `req.query.sort` is going to be a constant, and `sort` is a constant. `let` is ideal for when you have to share scope, or modify a single reference from various contexts.